modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/address/storage/AddressStorage.scala

```diff
@@ -4,6 +4,7 @@ import org.tessellation.schema.address.Address
 import org.tessellation.schema.balance.Balance
 
 trait AddressStorage[F[_]] {
+  def getState(): F[Map[Address, Balance]]
```

Comment:
imo it should be `def getState: F[Map[Address, Balance]]`

Reply:
Without the parenthesis, right.